ATTACK/dEpReCaTeD/Sur40ToImg.cs

```diff
@@ -0,0 +1,53 @@
+using System;
```

Comment:
What are these dEpReCaTeD files? Should they be deleted or are they going to be used?

Reply:
I deleted the files that should not be there anymore